packages/web-components/src/badge/fixtures/badge.html

```diff
@@ -8,25 +8,13 @@
 </style>
 <h1>Badge</h1>
 <h4>Default</h4>
-<fluent-badge>
-  Badge
-</fluent-badge>
+<fluent-badge> Badge </fluent-badge>
```

Comment:
It seems like this space should be trimmed, but I don't know if there's a setting for that.

Reply:
Yeah, that's really odd. May be because of the line length adjustments. I'll update manually and then we can keep an eye on it